Source/DafnyRuntime/DafnyRuntimeRust/src/lib.rs

```diff
@@ -9,6 +16,346 @@ pub use num::NumCast;
 pub use num::Zero;
 pub use itertools;
 
+// An atomic box is just a RefCell in Rust
+pub type SizeT = usize;
+
+/******************************************************************************
+ * Sequences
+ ******************************************************************************/
+
+// Three elements
+// ArraySequence(var isString: bool, nodeCount: SizeT, immutable_array)
+// ConcatSequence(var isString: bool, nodeCount: SizeT, left: Sequence, right: Sequence)
+// LazySequence(length: BigInteger, box: RefCell<Rc<Sequence<T>>>)
+// We use the named version using {...}, and use snake_case format
+
+// The T must be either a *const T (allocated) OR a Reference Counting (immutable)
+
+#[allow(dead_code)]
```

Comment:
Where does this occur and how hard is it to avoid?

Reply:
I removed all these annotations in the new PR. They were there because, contrary to Dafny, functions are private to the module by default, so if they are not prefixed by "pub", then it's dead code. By adding "pub" on every function like of Dafny (which I had done since), I now realized that this annotation of allowing dead code is no longer used, so I removed the 13 of them.